generated/nidcpower/nidcpower/lcr_measurement.py

```diff
@@ -0,0 +1,229 @@
+            AC compliance at the time the measurement was taken.
+
+        unbalanced (bool): Indicates whether the output was unbalanced at the time the
+            measurement was taken.
+    """
+    LCR = namedtuple(typename="LCR", field_names=("inductance", "capacitance", "resistance"))
+    InCompliances = namedtuple(typename="InCompliances", field_names=("dc", "ac"))
+
+    _lcr_measurement_field_metadata = [
+        # field_name           label(s)
+        ("channel"           , "Channel"                                                             ),  # noqa: E202,E203
+        ("vdc"               , "DC voltage"                                                          ),  # noqa: E202,E203
+        ("idc"               , "DC current"                                                          ),  # noqa: E202,E203
+        ("stimulus_frequency", "Stimulus frequency"                                                  ),  # noqa: E202,E203
+        ("ac_voltage"        , "AC voltage"                                                          ),  # noqa: E202,E203
+        ("ac_current"        , "AC current"                                                          ),  # noqa: E202,E203
+        ("z"                 , "Impedance"                                                           ),  # noqa: E202,E203
+        ("series_lcr"        , ("Series inductance", "Series capacitance", "Series resistance")      ),  # noqa: E202,E203
+        ("parallel_lcr"      , ("Parallel inductance", "Parallel capacitance", "Parallel resistance")),  # noqa: E202,E203
+        ("d"                 , "Dissipation factor"                                                  ),  # noqa: E202,E203
+        ("measurement_mode"  , "Measurement mode"                                                    ),  # noqa: E202,E203
+        ("in_compliances"    , ("DC in compliance", "AC in compliance")                              ),  # noqa: E202,E203
+        ("unbalanced"        , "Unbalanced"                                                          ),  # noqa: E202,E203
+    ]
+
+    def __init__(self, data):
+        """LCRMeasurement
+
+        Creates and returns an LCRMeasurement object.
+
+        Args:
+            data (struct_NILCRMeasurement): The LCR measurement ctypes object returned by the driver.
+        """
+        self.channel = ""
+        self.vdc = data.vdc
+        self.idc = data.idc
+        self.stimulus_frequency = data.stimulus_frequency
+        self.ac_voltage = complex(data.ac_voltage_real, data.ac_voltage_imaginary)
+        self.ac_current = complex(data.ac_current_real, data.ac_current_imaginary)
+        self.z = complex(data.z_real, data.z_imaginary)
+        self.series_lcr = LCRMeasurement.LCR(
+            inductance=data.ls, capacitance=data.cs, resistance=data.rs
+        )
+        self.parallel_lcr = LCRMeasurement.LCR(
+            inductance=data.lp, capacitance=data.cp, resistance=data.rp
+        )
+        self.d = data.d
```

Comment:
since you are keeping this around as a member, consider not storing any of the others (self.vdc, self.idc, etc) since they can be extracted or computed on demand from self.d.

Also, avoid abbreviations and call it self.data

Reply:
Nope, `data` is not stored as a member. `self.d` is not `data`, instead it is referring to dissipation factor, and it cannot be used to derive the other fields.